lib/portlayer/portlayer.go

```diff
@@ -14,8 +14,41 @@
 
 package portlayer
 
-import "github.com/vmware/vic/lib/portlayer/storage"
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/vic/lib/metadata"
+	"github.com/vmware/vic/lib/portlayer/storage"
+	"github.com/vmware/vic/pkg/vsphere/extraconfig"
+	"github.com/vmware/vic/pkg/vsphere/session"
+	"golang.org/x/net/context"
+)
+
+var Config metadata.VirtualContainerHostConfigSpec
 
 type API interface {
 	storage.ImageStorer
 }
+
+func Init(ctx context.Context, sess *session.Session) error {
+	source, err := extraconfig.GuestInfoSource()
+	if err != nil {
+		return err
+	}
+
+	extraconfig.DecodeWithPrefix(source, &Config, "guestinfo.vch")
```

Comment:
Hmm can we compare it against empty inferface? We are using Config.Networks just a line below and I think it will panic if decode fails for some reason.